specs/mediawiki/v1/content.yaml

```diff
@@ -62,12 +62,43 @@ paths:
       x-backend-request:
         uri: /{domain}/sys/page_revisions/page/
 
+  /{module:page}/title/{title}:
+    get:
+      tags:
+        - Page content
+      description: >
+        Returns the latest revision of the title
+
+        Stability: [experimental](https://www.mediawiki.org/wiki/API_versioning#Experimental)
+      produces:
+        - application/json
+      parameters:
+        - name: title
+          in: path
+          description: The page title.
+          type: string
+          required: true
+      responses:
+        '200':
+          description: The queriable list of page revisions.
```

Comment:
`/page/title/{title}/` (notice the trailing slash) returns that. This one should return the latest it knows about - either the latest in storage or the latest known by MW if the `Cache-Control: no-cache` header is provided (this is probably a valuable piece of information to clients, so it'd be good to put a note in the description)